### ISSUE TITLE
Question: Input Capture MACOS? and Hyprland 0.42 compatible?

### ISSUE THREAD

**Issue by prestonr83** [feschber/lan-mouse#192]
This just a question but I pulled latest from repo to build the MacOS client so I could remove the monitor restriction since I run dual monitors and I also had to build with higher pixel count for border for Hyprland Arch PC.

Initially I was set to make the Arch PC the server but to my surprise once I got everything connected the inputs from MacOS also transfer over and work on the Arch PC. 

Is this a new feature that is just implemented on main and not in a release yet?

Another question with Hyprland 0.42 dropping Wlroots is this going to affect compatibility? 

This is an awesome project you go got going. I have been pulling my hair out trying to get this to work between Hyprland and MacOS and yours was the only solution that works and so far works great. Just sent 5 bucks your way. Can't wait till there is encryption support.

**Comment by feschber** [comment]
Yes I recently merged the PR for MacOS. There are still some issues to iron out but it mostly works. I'm planning to make a new release soon.
 Thank you for the donation! Appreciate it!

**Comment by rodalpho** [comment]
Ahh, I was wondering why my mouse kept getting stuck on the mac side. Looking forward to the new release!

**Comment by broskees** [comment]
NVM, I got it working by running the macos binary as sudo
